pkg/cluster/cluster.go

```diff
@@ -172,7 +173,7 @@ func (c *Cluster) setStatus(status acidv1.PostgresStatus) {
 }
 
 func (c *Cluster) isNewCluster() bool {
-	return c.Status == acidv1.ClusterStatusCreating
+	return c.Status == acidv1.PostgresStatus{PostgresClusterStatus: acidv1.ClusterStatusCreating}
```

Comment:
Is it appropriate to compare the entire `PostgresStatus` struct here? Should it instead be looking only at `PostgresClusterStatus`?

```suggestion
	return c.Status.PostgresClusterStatus == acidv1.ClusterStatusCreating
```

Reply:
Thanks for your review. Your are right, should be a simple string comparison.